src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/Hubs/ChatHub.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+
+namespace Company.WebApplication1
+{
+    public class ChatHub : Hub
+    {
+        public Task SendMessageToGroup(string groupName, string message)
+        {
+            return Clients.Group(groupName).SendAsync("Send", $"{Context.ConnectionId} ({groupName}): {message}");
```

Comment:
Don't send connection ids to the client. This isn't secure and you can easily spoof other connections with these.

Reply:
Yeah, we do this in samples and definitely shouldn't.